src/analysis/ref_mode.rs

```diff
@@ -27,7 +27,11 @@ impl RefMode {
             ..
         }) = env.config.objects.get(&tid.full_name(&env.library))
         {
-            return ref_mode;
+            if direction == library::ParameterDirection::In {
+                return ref_mode;
+            } else {
+                return RefMode::None;
```

Comment:
It would also be for Out parameters (and InOut I guess, but we don't really support them).